### DISCUSSION TITLE
0.2.3-stable has been released!

### DISCUSSION THREAD

**Discussion by mihe** [godot-jolt/godot-jolt#452] [Announcements]
This release supports [**Godot 4.0.3**](https://github.com/godotengine/godot/releases/tag/4.0.3-stable).

## Known major issues

- Extension classes cause exception when used from C# ([#270](https://github.com/godot-jolt/godot-jolt/issues/270))
- The `move_and_collide` parameter `recovery_as_collision` is always `true` ([#337](https://github.com/godot-jolt/godot-jolt/issues/337))

## Changelog

These are the notable changes that have been made since [0.2.2-stable](https://github.com/godot-jolt/godot-jolt/releases/tag/v0.2.2-stable) was released. You can also find a list of all the commits [here](https://github.com/godot-jolt/godot-jolt/compare/v0.2.2-stable...v0.2.3-stable).

Any breaking changes are denoted with ⚠️.

### Fixed

- Fixed issue where bodies would transform in unintuitive ways when attached to a rotated joint.
- Fixed issue where bodies would sometimes transform in unintuitive ways when attached to a `Generic6DOFJoint` that used both linear and angular limits.
- Fixed issue where setting the limits of a `SliderJoint3D` to the same value would make it free instead of fixed.
- Fixed issue where you could still rotate a `RigidBody3D` slightly while using `lock_rotation`.
- Fixed issue where friction would be applied more on one axis than the other.

<hr /><em>This discussion was created from the release <a href='https://github.com/godot-jolt/godot-jolt/releases/tag/v0.2.3-stable'>0.2.3-stable</a>.</em>

**Comment by phelioz** [comment]
Great as always, grymt! :)

**Reply by mihe** [reply]
Tackar tackar! :)
